src/components/Navigator/NavGroup/NavGroup.tsx

```diff
@@ -67,7 +66,7 @@ function NavGroup({
             <Icon
               name={leftIcon}
               size={IconSize.S}
-              color={leftIconColor}
+              color={active ? undefined : 'txt-black-dark'}
```

Comment:
```suggestion
              color={active ? 'bgtxt-blue-normal' : 'txt-black-dark'}
```
`undefined`로 상속받은 스타일을 사용해도 좋지만 이게 가독성이 좀 더 좋지 않을까 생각해봅니다.

Reply:
가독성은 조금 떨어지지만 상속을 받아야 item 한 곳에서 active color를 관리할 수 있어 말씀해주신 부분과 약간의? trade off가 있는 것 같긴합니다!
가독성을 개선할 수 있는 방법으로 가는 게 좋을까요??

Reply:
둘 다 큰 차이가 없을 것 같아서, 생각해보고 결정해주시면 될 것 같습니다

Reply:
어차피 nonActiveStyle의 color와 leftIcon의 color가 다르긴하네요!
leftIconColor는 별도로 지정해주는 게 더 나아보입니다. 반영하겟습니다~